lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
```

Comment:
I don't think `frozen_string_literal` is required here as it's evaluated in ERB.

Reply:
This was created by initializing gem too. I can delete this from this file, but this is lib file